### PR TITLE
changes to 42 series wasmd (CI & testing)

### PR DESCRIPTION
**Pull request by faddat**
In setting up a rigorous ci system, I discovered a usage of math.rand instead of crypto.rand. 

I don't know how much it matters, but gosec (the informal fork) flagged it, so here's a PR :)

### CODE REVIEW

File: x/wasm/types/test_fixtures.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func randBytes(n int) []byte {
 	r := make([]byte, n)
 	rand.Read(r)
 	return r
-}
+}beta2
```
Comment (ethanfrey) [inline x/wasm/types/test_fixtures.go]: I bet that was a typo

**Review by ethanfrey — COMMENTED** [review]
Hmmm... for test cases, goes this matter?

I guess it gets picked up as `test_fixtures.go` is not technically test code

File: x/wasm/types/test_fixtures.go (hunk 1 above)
Comment (faddat) [reply x/wasm/types/test_fixtures.go]: yep

**Comment by faddat** [comment]
Before I start working on forking again, I will omnibus all of my changes for "baseline ci" here.

File: x/wasm/types/test_fixtures.go (hunk 1 above)
Comment (faddat) [reply x/wasm/types/test_fixtures.go]: re: does it matter?

honestly when I do stuff sometimes I just choose to be a slave to the linters

**Pull request closed without merging**